src/KeyVault/Microsoft.Azure.KeyVault/Customized/KeyVaultClientExtensions.cs

```diff
@@ -413,18 +413,19 @@ public static partial class KeyVaultClientExtensions
         /// <summary>
         /// Updates a certificate version.
         /// </summary>
-        /// <param name="certificateIdentifier">The URL for the certificate.</param>             
+        /// <param name="certificateIdentifier">The URL for the certificate.</param>      
+        /// <param name='certificatePolicy'>The management policy for the certificate.</param>       
         /// <param name="certificateAttributes">The attributes of the certificate (optional)</param>
         /// <param name="tags">Application-specific metadata in the form of key-value pairs</param>
         /// <param name="cancellationToken">Optional cancellation token</param>
         /// <returns>The updated certificate.</returns>
-        public static async Task<CertificateBundle> UpdateCertificateAsync(this IKeyVaultClient operations, string certificateIdentifier, CertificateAttributes certificateAttributes = null, IDictionary<string, string> tags = null, CancellationToken cancellationToken = default(CancellationToken))
+        public static async Task<CertificateBundle> UpdateCertificateAsync(this IKeyVaultClient operations, string certificateIdentifier, CertificatePolicy certificatePolicy = default(CertificatePolicy), CertificateAttributes certificateAttributes = null, IDictionary<string, string> tags = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
>UpdateCertificateAsync [](start = 52, length = 22)

Is there a similar method to just update the certificate policy. Certificate Policy is a child resource and can be updated by itself.